src/sinkhorn.jl

```diff
@@ -182,19 +182,20 @@ optimal `dual_potential`'s of `a` and `b`.
 """
 sinkhorn_divergence!(
     D::AbstractDivergence,
+    C,
     a::DiscreteMeasure,
     b::DiscreteMeasure,
     ϵ = 1e-1;
     kwargs...,
-) = _sinkhorn_divergence!(D, a, b, ϵ; kwargs...)
+) = _sinkhorn_divergence!(D, C, a, b, ϵ; kwargs...)
 
```

Comment:
```suggestion

sinkhorn_divergence!(
    D::AbstractDivergence,
    C::AbstractMatrix,
    a::DiscreteMeasure,
    b::DiscreteMeasure,
    ϵ = 1e-1;
    kwargs...,
) = throw(ArgumentError("Must pass a cost function `C`, not a cost matrix."))

```
I think we should add this, because otherwise you can get silently wrong answers with the current implementation.